casr/src/bin/casr-core.rs

```diff
@@ -135,8 +125,8 @@ fn main() -> Result<()> {
             .with_context(|| format!("Couldn't read core: {}", core_path.display()))?;
         let mut report = CrashReport::new();
 
-        if matches.is_present("executable") {
-            let executable_path = PathBuf::from(matches.value_of("executable").unwrap());
+        if matches.contains_id("executable") {
+            let executable_path = PathBuf::from(matches.get_one::<String>("executable").unwrap());
```

Comment:
path

Reply:
Should be string, because of online mode.